modules/i18n/input-method/default.nix

```
@@ -0,0 +1,67 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.i18n.inputMethod;
+
+  gtk2_cache = pkgs.runCommandLocal "gtk2-immodule.cache" {
```

Ideally variables should be camel cased:
```suggestion
  gtk2Cache = pkgs.runCommandLocal "gtk2-immodule.cache" {
```

Same with `gtk3_cache` below.